reana/reana_dev/run.py

```diff
@@ -281,7 +348,14 @@ def run_ci(
 )
 @run_commands.command(name="run-example")
 def run_example(
-    component, workflow_engine, file, timecheck, timeout, parameters, options
+    component,
+    workflow_engine,
+    compute_backend,
+    file,
+    timecheck,
+    timeout,
+    parameters,
+    options,
```

Comment:
Please add new options `--submit-only` which will run everything up to starting workflows.

Later we'll add `--check-only` which will verify statuses and files.

In this way the big job matrix can be sort of parallel...